### PR TITLE
[WIP] Refactoring rosie to create the core module

### PR DESCRIPTION
**Pull request by anaschwendler**
After talking to @cuducos, we started creating the core module that will make rosie generic.
By now what is done:
- Structure created
- Moved one classifier to the core module, ensuring that rosie keep working fine

What need to be done:
- Move core methods from `__init__.py` that is in chamber_of_deputies to `__init__.py` of core module
- Create `settings.py` with the definition of the classifiers used in chamber_of_deputies and `adapters.py` with the dataset_keys to chamber_of_deputies
- Prepare `rosie.py` to run each module separate (ex.: `python rosie.py run chamber_of_deputies`)

### CODE REVIEW

**Comment by Irio** [comment]
Where are you thinking to put the `settings.py` files?

**Comment by anaschwendler** [comment]
@Irio every `settings.py` file will be put in its module folder, as described in [this document](https://gist.github.com/cuducos/c98a91a9447b9404abd3b449daba50ec) made by @cuducos.

**Comment by cuducos** [comment]
> every `settings.py` file will be put in its module folder

That's completely true, but I'd like to expand on it: each module is a different app, meaning that each module implements Rosie for a certain purpose (let's say, Chamber of Deputies, Senate, than cities etc.). So each one of these Rosie applications has its own `settings.py` to specify which classifiers to use (probably all from their own module, but maybe some from the core module) and how to use them (e.g. should it validate CNPJ *and* CPF or *only* CNPJ?). Probably there will be some adapters too (e.g. mapping unique identifiers — `document_id` in the case o CEAP — and total values — `total_net_value` in the case of CEAP).

At least this is the idea by now ; )

File: rosie/chamber_of_deputies/settings.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'over_monthly_subquota_limit': MonthlySubquotaLimitClassifier,
+    'suspicious_traveled_speed_day': TraveledSpeedsClassifier,
+    'invalid_cnpj_cpf': InvalidCnpjCpfClassifier,
+    'election_expenses': ElectionExpensesClassifier,
+    'irregular_companies_classifier': IrregularCompaniesClassifier
+}
+
+UNIQUE_IDS = ['applicant_id', 'year', 'document_id']
+
+VALUE = 'total_net_value'
```
Comment (Irio) [inline rosie/chamber_of_deputies/settings.py]: This list of columns will quickly grow large. `total_net_value`, `cnpj_cpf`, `subquota_id` already have different names in Chamber of Deputies and Federal Senate. I'd rather prefer to keep in the `settings.py` a dictionary with dataset_column -> column expected by Rosie, as in:

```python
COLUMNS = {
  'value': 'total_net_value',
  'company_id': 'cnpj_cpf',
}
```

In the case of this example, I suggest to have `rosie.chamber_of_deputies.adapter.Adapter` renaming the dataset columns and the classifiers look for columns named `'value'` and `'company_id'`.

**Pull request merged**